### PR TITLE
feat(GridFormTextArea): add rows to text-area field

### PR DESCRIPTION
**Pull request by dreamwasp**
### Overview

Add the rows prop to GridFormTextArea

### PR Checklist

- [x] Related to designs:
- [x] Related to JIRA ticket: [web-2447]
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change
- [x] This PR includes testing instructions tests for the code change
- [x] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

* View the larger TextArea in the "With tooltip" section of the GridForm story
#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/36478)  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/3578)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-3578)   |

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@55.5.0-alpha.189d61.0<br/>@codecademy/gamut-kit@0.6.345-alpha.189d61.0<br/>@codecademy/styleguide@66.3.0-alpha.189d61.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://650494ebba445e08553cce65--gamut-preview.netlify.app>https://650494ebba445e08553cce65--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/650494ebba445e08553cce65>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@55.5.0-alpha.672a74.0<br/>@codecademy/gamut-kit@0.6.345-alpha.672a74.0<br/>@codecademy/styleguide@66.3.0-alpha.672a74.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6504a00d851b27122d2193f6--gamut-preview.netlify.app>https://6504a00d851b27122d2193f6--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6504a00d851b27122d2193f6>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6509d3c3636e3d24bd1f8798--gamut-preview.netlify.app>https://6509d3c3636e3d24bd1f8798--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6509d3c3636e3d24bd1f8798>Deploy Logs</a></p>

File: packages/gamut/src/GridForm/types.ts [hunk 1]
```diff
@@ -115,12 +115,14 @@ export type GridFormFileField = BaseFormField<FileList> & {
...
-  label: React.ReactNode;
-  placeholder?: string;
-  validation?: RegisterOptions;
-  type: 'textarea';
-};
+export type GridFormTextAreaField = BaseFormField<string> &
+  Pick<TextAreaProps, 'rows' | 'placeholder'> & {
+    label: React.ReactNode;
+    // placeholder?: string;
+    // rows?: number;
```
Comment (aresnik11) [inline packages/gamut/src/GridForm/types.ts:122]: should these be uncommented? or removed?

**Review by aresnik11 — APPROVED** [review]
LGTM! ✨

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://650cacdd7e223731bebfd9fe--gamut-preview.netlify.app>https://650cacdd7e223731bebfd9fe--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/650cacdd7e223731bebfd9fe>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@55.5.0-alpha.30133e.0<br/>@codecademy/gamut-kit@0.6.345-alpha.30133e.0<br/>@codecademy/styleguide@66.3.0-alpha.30133e.0<br/></pre>

**Pull request merged**